sql/derived_tables/items_ext.sql

```diff
@@ -3,9 +3,10 @@ DROP TABLE IF EXISTS local.items_ext;
 -- Create an extended items table that includes the name for in
 -- transit destination service point, item damaged status, material
 -- type call number type, permanent loan type, permanent location,
--- temporary loan type, temporary location.
-CREATE TABLE local.items_ext AS
-WITH items AS (
+-- temporary loan type, temporary location, created_date,
+-- description_of_pieces, status_date, status_name, holdings_id
+
+CREATE TABLE local.items_ext AS WITH items AS (
```

Comment:
Interesting that the WITH keyword has been formatted on the same line with CREATE TABLE